### ISSUE TITLE
Saving cloud home crashes to local 100% of the time

### ISSUE THREAD

**Issue by TiamatKefir** [Yellow-Dog-Man/Resonite-Issues#373]
### Describe the bug?

Saving world changes to my cloud home crashes me to local, but the world successfully updates and saves the changes.

### To Reproduce

Save cloud home.
Crash to local.

### Expected behavior

Save cloud home.
Stay in cloud home.

### Resonite Version Number

2023.10.19.620 and below

### What Platforms does this occur on?

Windows

### What headset if any do you use?

Vive and Desktop

### Log Files

I don't feel comfortable sending logs to a publicly viewable forum, but would be more than happy to DM someone with the logs.

**Comment by TiamatKefir** [comment]
Yes, I understand, but there is sensitive information in the logs, so if it's possible to DM or redact said info from the logs I would prefer that.

**Comment by TiamatKefir** [comment]
Here is a fresh launch log, please let me know if I need to do anything else.
[LAPTOP - 2023.12.13.87 - 2023-12-13 19_54_35(1).log](https://github.com/Yellow-Dog-Man/Resonite-Issues/files/13667149/LAPTOP.-.2023.12.13.87.-.2023-12-13.19_54_35.1.log)

**Comment by TiamatKefir** [comment]
I've got a handful of things like the Event Calendar, SSTV, Oscilloscope, potions box, and then just the stock cloud home items.

**Comment by shiftyscales** [comment]
Is this issue still relevant, @TiamatKefir? If so, did you determine what item was responsible for the crashes?
